sprint0/sprint0/Enemies/SpawnCloud.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+
+namespace sprint0
+{
+    public class SpawnCloud : IEnemy
```

Comment:
highly prefer if this could be an IEffect. so like when we want to spawn an enemy, we make the Ieffect that then makes the IEnemy or ISprite